src/app/target-curve-points/target-curve-points.component.ts

```diff
@@ -69,4 +73,12 @@ export class TargetCurvePointsComponent {
       console.warn('Cannot paste', e)
     }
   }
+
+  private sortPoints() {
+    this.curvePoints.sort((a, b) => {
```

Comment:
Not the most efficient sort here, as we're re-parsing the string every time a comparison is made.

It's a bigger change that I'd like to make separately, but ideally we'd do all this parsing on load and store things in memory as numeric values, re-serializing to the text value on export if/when needed

Reply:
yea, I would rewrite this component to work with numbers only, and 'stringify' on export.
It would also make data validation easier.

The only risk here is that the Original iOS app has numbers with greater accuracy then browser Javascript Number type.

So we can try but it'll require some time for rewriting and testing. So not sure if it's worth it?

Reply:
Agree it's probably not worth prioritizing/focusing on vs other functionality.  I just hate writing code I know is incredibly non-performant :)

Hadn't thought about the resolution of numbers across the platforms.  It probably doesn't matter if we lose some precision but better to not mess with it if not needed